### PR TITLE
Update Readme file

### PR DESCRIPTION
**Pull request by Shrey1508**
Update readme file with proper context
what we are automating(amazon)
which node version we are using

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# **Amazon Login Page**
+# **Amazon Login Page and Add to Cart**
 
-_Amazon login fuctionality is implementing_
+_Amazon login Product Add To Cart fuctionality is implementing_
```
Comment (h2grover) [inline README.md:3]: why you are using `_` here? Please remove it.

Comment (Shrey1508) [reply README.md:3]: ok

Comment (Shrey1508) [reply README.md]: Removed _

**Comment by hrk15** [comment]
@Shrey1508 Merge this PR.

**Pull request merged**